browser/components/customizableui/content/panelUI.js

```diff
@@ -276,6 +276,15 @@ const PanelUI = {
     switch (aEvent.type) {
       case "popupshowing":
         updateEditUIVisibility();
+        try {
+          if (!Services.prefs.getBoolPref("browser.restart_menu.showpanelmenubtn")) {
+            document.getElementById("appMenu-restart-button").hidden = true;
+          } else {
+            document.getElementById("appMenu-restart-button").hidden = false;
+          }
+        } catch (e) {
+          throw new Error("We're sorry but something has gone wrong with 'browser.restart_menu.showpanelmenubtn'" + e);
```

Comment:
This error keeps getting thrown when accessing the hamburger menu on macOS. Tested with both restart button enabled/disabled.

`JavaScript error: chrome://browser/content/customizableui/panelUI.js, line 286: Error: We're sorry but something has gone wrong with 'browser.restart_menu.showpanelmenubtn'TypeError: document.getElementById(...) is null`

Reply:
Ah, I see the problem, I added appMenu-restart-button inside of `#ifndef XP_MACOSX`, so it's not available for Mac.

Reply:
Cool, thanks for the fix.